Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -978,6 +990,50 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    /// SwifterSwift: Overload Swift's 'contains' operator for matching regex
+    ///
+    /// - Parameter lhs: String to check on regex.
+    /// - Parameter rhs: Regex to match against.
+    /// - Returns: true if there is at least one match for the regex in the string.
+    static func ~= (lhs: String, rhs: NSRegularExpression) -> Bool {
+        let range = NSRange(location: 0, length: lhs.utf16.count)
+        return rhs.firstMatch(in: lhs, options: [], range: range) != nil
+    }
+    #endif
+
+    #if canImport(Foundation)
+    /// SwifterSwift: Returns a new string in which all occurrences of a regex pattern in a specified range of the receiver are replaced by the template.
+    /// - Parameter pattern: Regex pattern to replace.
+    /// - Parameter template: The regex template to replace the pattern.
+    /// - Parameter searchRange: The range in the receiver in which to search.
+    /// - Returns: A new string in which all occurrences of regex pattern in searchRange of the receiver are replaced by template.
+    func replacingOccurrences(of pattern: String, with template: String,
+                              range searchRange: Range<String.Index>? = nil) -> String {
```

Comment:
```suggestion
    func replacingOccurrences(of pattern: String,
                              with template: String,
                              range searchRange: Range<String.Index>? = nil) -> String {
```
Did you run `swiftformat`? 😛

Reply:
Indeed, pre-commit hook installed! 🎉

Reply:
And it doesn't put each parameter on a new line? That's weird, no?

Reply:
It seems to take a simple approach that wraps if lines are too long. Maybe it can be configured. If it has to wrap I'd prefer one-per-line. Good news is that it will preserve newlines if you do this manually.